### PR TITLE
Clean up grammar in README

### PR DESCRIPTION
**Pull request by kevinburkeshyp**
Makes some of the sentences read more like idiomatic English. Try to document
some of the function signatures in the headers. Add syntax highlighting for two
of the examples.

### CODE REVIEW

**Comment by kevinburkeshyp** [comment]
In addition - is this actually an assertion? `promised('abc').should.finally.be.a.String`. Should that be `promised('abc').should.finally.be.a(String)`

**Comment by btd** [comment]
Hi, thanks for fixes.
`promised('abc').should.finally.be.a.String` is not an assertion because `.String` method not called. It should be `.String()`. Your second version will not work with should.js (that old syntax was removed in initial version - it will work maybe only in chai).
Can you pls also fix title for `.rejectedWith(error)`?

File: README.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ it('should be allow to check if promise rejected', function() {
 
 ## .rejectedWith
```
Comment (benbuckman) [inline README.md]: Suggestion, `.rejectedWith(err)` to match the `.fulfilledWith(value)` change above?
Thanks

Comment (benbuckman) [inline README.md]: Actually that's not the real signature. It's confusing – maybe best to just link to it – http://shouldjs.github.io/#assertion-throw

**Comment by kevinburkeshyp** [comment]
Pushed fixes in 2ebbcbe.

**Comment by benbuckman** [comment]
:+1:

**Comment by btd** [comment]
Thanks, guys :+1:

**Pull request merged**